README.md

```diff
@@ -129,6 +130,9 @@ The sidebar consists of multiple widgets. Widgets can be enabled individually us
 
 Some widget respect optional configuration. Have a look at the `[Params.widgets]` and `[Params.widgets.social]` sections in the example configuration above.
 
+### Logo
+
+**Mainroad** can be customized with your own logo in the site header. You just have to provide a `logo=path/to/your/logo.svg` stanza in your configuration file (see above). Mainroad will resize the logo depending on the display size of the client. Thus, we expect you to provide the logo in a vector format (`.svg`).
```

Comment:
Specify max-height resize value 80px (5rem is equal to 80px in our case) in README.md

Reply:
Are you sure? Adding pixels to the description motivates users to provide non-vector graphics. As soon as we redesign things break.

Reply:
> Are you sure? Adding pixels to the description motivates users to provide non-vector graphics. 

I think we made it clear about SVG:

> Thus, we expect you to provide the logo in a vector format (`.svg`)